publication-model/src/main/java/no/unit/nva/model/instancetypes/artistic/film/realization/OtherRelease.java

```diff
@@ -12,7 +14,7 @@
 import nva.commons.core.JacocoGenerated;
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
-public class OtherRelease implements MovingPictureOutput, UnconfirmedPublisherMigrator {
+public class OtherRelease implements MovingPictureOutput, UnconfirmedPublisherMigrator, Published {
```

Comment:
I introduce the concept of "Published" for things that are published.

This allows me to use polymorphism to get the channel uris.